### PR TITLE
Upgrade guide for saga persisters and saga finders

### PR DESCRIPTION
**Pull request by danielmarbach**
Relates to https://github.com/Particular/NServiceBus/pull/2794

### CODE REVIEW

**Comment by danielmarbach** [comment]
@Particular/documentation please review

File: nservicebus/upgrades/5to6.md [hunk 1]
```diff
@@ -99,6 +99,8 @@ Configuration should be easy to apply to your code and easy to manipulate by han
 
 NServiceBus will automatically make correlated saga properties unique without the need for a explicit `[Unique]` attribute to be used. You can safely remove the attribute from your saga data types.
 
+### Saga persisters & finders
+Saga persisters `ISagaPersister` and finders `IFindSagas` introduced a new method parameter `SagaPersistenceOptions`. The options allow to access the saga metadata and float countext bound objects into the saga persisters and finders. Therefore enabling to access everything which is added to the context during the pipeline execution.
```
Comment (SimonCropp) [inline nservicebus/upgrades/5to6.md]: > Therefore enabling to access everything

That needs rewording

**Comment by danielmarbach** [comment]
@SimonCropp did a small rewording.

File: nservicebus/upgrades/5to6.md [hunk 2]
```diff
@@ -99,6 +99,8 @@ Configuration should be easy to apply to your code and easy to manipulate by han
 
 NServiceBus will automatically make correlated saga properties unique without the need for a explicit `[Unique]` attribute to be used. You can safely remove the attribute from your saga data types.
 
+### Saga persisters & finders
+Saga persisters `ISagaPersister` and finders `IFindSagas` introduce a new method parameter `SagaPersistenceOptions`. These options give access to the saga metadata and pipeline context within saga persisters and finders. This enables custom saga persisters and custom saga finders to have access to everything which is added to the context during message pipeline exectuion.
```
Comment (SimonCropp) [inline nservicebus/upgrades/5to6.md]: would a before after sample be useful here? or at least a link to the existing doco that shows both versions?

**Pull request merged**